release/src/main/scripts/build_release_candidate.sh

```diff
@@ -56,12 +56,19 @@ read USER_GITHUB_ID
 
 USER_REMOTE_URL=git@github.com:${USER_GITHUB_ID}/beam-site
 
+echo "================Listing all GPG keys================="
+gpg --list-keys --keyid-format LONG --fingerprint --fingerprint
+echo "Please copy the public key which is associated with your Apache account:"
+
+read SIGNING_KEY
```

Comment:
Should we check that this is matching: user.signingkey (from: https://github.com/apache/beam/blob/master/release/src/main/scripts/preparation_before_release.sh#L48) or usage from other scripts?

Reply:
Probably yes.

But we did not check before [1], so I did not bother to implement this. As it probably would require to keep some state across scripts. Currently this is left to manual release verification.

As I tend to assume that these script need some rework anyway, I restricted the scope of this PR to a minimal viable solution to get release enabled on gradle5.

[1] There is no check on signing key set in git config against key put into KEYS file nor against default key used for signing artefacts.

Reply:
Sounds good. Do you mind add a JIRA todo comment here to clean this up in all scripts?